app/js/cartogram.js

```diff
@@ -106,7 +107,7 @@
               sizeError = Math.max(area, desired) / Math.min(area, desired);
             sizeErrorsTot += sizeError;
             sizeErrorsNum++;
-            //console.log(o.id, "@", j, "area:", area, "value:", v, "->", desired, radius, mass, sizeError);
+            console.log(o.id, "@", j, "area:", area, "value:", v, "->", desired, radius, mass, sizeError);
             return {
```

Comment:
remove
We're trying to remove all unnecessary console logs to prepare for release